html-tables/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Tables</title>
+    <body>
+      <h1>Student Grade Table</h1>
+      <table>
+        <thead>
+          <tr>
+            <th>Student Name</th>
+            <th>Course</th>
+            <th>Grade</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Grace Hopper</td>
+            <td>Computer Science</td>
+            <td>100</td>
+          </tr>
+          <tr>
+            <td>Alan Turing</td>
+            <td>Mathematics</td>
+            <td>100</td>
+          </tr>
+          <tr>
+            <td>
+              Douglas Crockford
+            </td>
+            <td>JSON</td>
+            <td>98</td>
+          </tr>
+          <tr>
+            <td>Tim Berners-Lee</td>
+            <td>HTML</td>
+            <td>9001</td>
+          </tr>
+        </tbody>
+          <tr>
+  <td>Linus Torvalds</td>
+  <td>Git</td>
+  <td>90</td>
+          </tr>
+
```

Comment:
this tr should be within the tbody. I would also adjust the indentation to keep it uniform with the rest of the table